htap_test.go

```diff
@@ -0,0 +1,102 @@
+package gosnowflake
+
+import (
+	"encoding/json"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func TestDeserializeOpaqueContext(t *testing.T) {
```

Comment:
you test deserialization and serialization in the test? maybe it should reflect it in the test name

also consider using Go specific names - decode and marshal?

Reply:
Done.